channeldb/db.go

```diff
@@ -276,6 +276,11 @@ var (
 	// channelOpeningState for each channel that is currently in the process
 	// of being opened.
 	channelOpeningStateBucket = []byte("channelOpeningState")
+
+	// initialChannelForwardingPolicy is the database bucket used to store the
```

Comment:
Commit message prefix should be `funding+channeldb:` then the text itself can just be `handle and persist channel fees` for example.